client/index.html

```diff
@@ -1,27 +1,81 @@
-    <script src="./assets/js/index.js"></script>
+  <header class="header-layout">
+      <div class="header-layout__title">My 배민</div>
+  </header>
+  <main class="main-page-layout">
+    <section class="main-section">
+      <div class="main-banner"></div>
+      <a class="login-btn">
+        <img class="login-btn__user" src="./assets/icons/user.png" alt="user_icon"/> 
+        <div class="login-btn__title">로그인 해주세요</div>
+        <img class="login-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+      <nav class="main-menu-layout">
+        <ul class="main-menu">
+          <a class="main-menu__item">
+            <img src="./assets/icons/point.png" alt="point_icon"/>
+            <span>포인트</span>
+          </a>
+          <a class="main-menu__item">
+            <img src="./assets/icons/coupon.png" alt="coupon_icon"/>
+            <span>쿠폰함</span>
+          </a>
+          <a class="main-menu__item">
+            <img src="./assets/icons/gift.png" alt="gift_icon"/>
+            <span>선물함</span>
+          </a>
+          <a class="main-menu__item">
+            <img src="./assets/icons/like.png" alt="like_icon"/>
+            <span>찜</span>
+          </a>
+          <a class="main-menu__item">
+            <img src="./assets/icons/order.png" alt="order_icon"/>
+            <span>주문내역</span>
+          </a>
+          <a class="main-menu__item">
+            <img src="./assets/icons/review.png" alt="review_icon"/>
+            <span>리뷰관리</span>
+          </a>
+        </ul>
+      </nav>
+    </section>
+    <section class="main-section campaign-section">
+      <a class="main-btn">
+        <div class="main-btn__title">일회용품 덜 쓰기, 함께해요!</div>
+        <img class="main-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+    </section>
+    <section class="main-section">
+      <a class="main-btn">
+        <div class="main-btn__title">배민페이 등록</div>
+        <div class="main-btn__brief">배민페이로 결제하면 최대 5.5% 배민포인트가 적립됩니다!</div>
+        <img class="main-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+      <a class="main-btn">
+        <div class="main-btn__title">가족계정 관리</div>
+        <div class="main-btn__brief">결제수단을 공유해 우리 가족의 끼니를 챙겨주세요.</div>
+        <img class="main-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+      <a class="main-btn">
+        <div class="main-btn__title">공지사항</div>
+        <img class="main-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+      <a class="main-btn">
+        <div class="main-btn__title">배민커넥트 지원</div>
+        <img class="main-btn__icon" style="transform: rotate(180deg)" src="./assets/icons/left.svg" alt="right_icon"/>
+      </a>
+    </section>
+  </main>
+  <script src="./assets/js/index.js"></script>
```

Comment:
추후에 스크립트 태그를 head 안에 넣고 defer 속성을 붙여도 좋을 것 같습니다~

```suggestion
  <script defer src="./assets/js/index.js"></script>
```